core/src/main/java/io/aiven/klaw/service/AclControllerService.java

```diff
@@ -523,11 +462,21 @@ public List<AclRequestsResponseModel> getAclRequestsForApprover(
 
     createdAclReqs = filterAclRequestsByTenantAndOrder(getCurrentUserName(), createdAclReqs, order);
 
-    return getAclRequestModelPaged(
-        updateCreatAclReqsList(createdAclReqs, tenantId, userDetails),
+    return Pager.getItemsList(
         pageNo,
         currentPage,
-        tenantId);
+        10,
+        updateCreatAclReqsList(createdAclReqs, tenantId, userDetails),
+        (pageContext, aclRequestsModel) -> {
+          aclRequestsModel.setAllPageNos(pageContext.getAllPageNos());
+          aclRequestsModel.setTotalNoPages(pageContext.getTotalPages());
+          aclRequestsModel.setCurrentPage(pageContext.getPageNo());
+          aclRequestsModel.setTeamname(
+              manageDatabase.getTeamNameFromTeamId(tenantId, aclRequestsModel.getTeamId()));
+          aclRequestsModel.setEnvironmentName(
+              getEnvDetails(aclRequestsModel.getEnvironment(), tenantId).getName());
```

Comment:
can we replace this getEnvDetails with the method defined in commonUtilsService.getEnvDetails

Reply:
yes, done